pex/util.py

```diff
@@ -213,3 +215,32 @@ def named_temporary_file(*args, **kwargs):
       yield fp
   finally:
     os.remove(fp.name)
+
+
+def iter_pth_paths(filename):
+  """Given a .pth file, extract and yield all inner paths. Largely lifted from site.py."""
+  dirname = os.path.dirname(filename)
+  known_paths = set()
+
+  try:
+    f = open(filename, 'rU')  # noqa
+  except IOError:
+    return
+
+  with f:
+    for line in f:
+      line = line.rstrip()
+      if not line or line.startswith('#'):
+        continue
+      elif line.startswith(('import ', 'import\t')):
+        try:
+          exec_function(line)
+          continue
+        except Exception:
+          # Defer error handling to the higher level site.py logic invoked at startup.
+          return
```

Comment:
Same here. We're not deferring error handling to `site.py` any more, since we've copy-pasted this out of startup code and into regular userland code.